### PR TITLE
BZ 1666367: correcting API usage

### PR DESCRIPTION
**Pull request by ousleyp**
https://bugzilla.redhat.com/show_bug.cgi?id=1666367

### CODE REVIEW

**Comment by stu-gott** [comment]
Looks good to me. Nice job!

**Review by davidvossel — CHANGES_REQUESTED** [review]
Looks good. I think I may have caught a few instances where volumeName still needs to be addressed though

When I run a grep filter, I still see some references to volumeName when I checkout your branch.
```
$ grep -r -n "volumeName" modules/*
modules/cnv_creating_vm.adoc:30:|disks: volumeName 
modules/cnv_pxebooting.adoc:96:    volumeName: registryvolume 
modules/cnv_template_vm_config.adoc:29:            volumeName: root
```

**Comment by ousleyp** [comment]
> Looks good. I think I may have caught a few instances where volumeName still needs to be addressed though
> 
> When I run a grep filter, I still see some references to volumeName when I checkout your branch.
> 
> ```
> $ grep -r -n "volumeName" modules/*
> modules/cnv_creating_vm.adoc:30:|disks: volumeName 
> modules/cnv_pxebooting.adoc:96:    volumeName: registryvolume 
> modules/cnv_template_vm_config.adoc:29:            volumeName: root
> ```

Oh, weird! I'm surprised I didn't see those when I grepped earlier. Thanks so much for checking.

Commit: https://github.com/openshift/openshift-docs/pull/13398/commits/14fae5625f62031869a6d129e5a9b6fde7a979ca

**Review by davidvossel — APPROVED** [review]
excellent :+1:

File: modules/cnv_creating_vm_web_console.adoc [hunk 1]
```diff
@@ -39,17 +39,17 @@ namespace where you want the VM to be created.
 
 . Next, select the provisioning source from the following options:
 [horizontal]
-*PXE*:: The virtual machine will be booted over the network. The network
-interface and logical network will be configured later in the
+*PXE*:: The virtual machine is booted over the network. The network
```
Comment (kalexand-rh) [inline modules/cnv_creating_vm_web_console.adoc]: s/is booted/boots

Comment (kalexand-rh) [reply modules/cnv_creating_vm_web_console.adoc]: You configure the network interface and logical network later...

File: modules/cnv_creating_vm_web_console.adoc [hunk 2]
```diff
@@ -39,17 +39,17 @@ namespace where you want the VM to be created.
...
+interface and logical network is configured later in the
 *Networking* tab.
 *URL*:: Provide the address of a disk image accessible from 
 {product-title}. RAW and QCOW2 formats are supported. Either 
-format can be compresed with *gzip* or *xz*.
-*Registry*:: Provide a container containing a bootable operating system in a
-registry accessible from {product-title}. A *registryDisk* 
-volume is ephemeral, and the volume will be discarded when the virtual machine 
+format can be compressed with *gzip* or *xz*.
```
Comment (kalexand-rh) [inline modules/cnv_creating_vm_web_console.adoc]: You can compress either format...

File: modules/cnv_creating_vm_web_console.adoc [hunk 3]
```diff
@@ -65,7 +65,7 @@ network using `NetworkAttachmentDefinition`.
 machine disks. If you want to attach a PVC from this screen, 
 it must already exist. 
 
-. Once you have created the VM, it will be visible under
+. After you have created the VM, you can find it listed under
```
Comment (kalexand-rh) [inline modules/cnv_creating_vm_web_console.adoc]: s/have created/create
s/you can find it/it is

File: modules/cnv_vm_storage_volume_types.adoc [hunk 4]
```diff
@@ -31,14 +31,14 @@ Attaches a disk containing the referenced cloud-init NoCloud data
...
+*containerDisk*::
 References an image, such as a virtual machine disk, that is stored in
 the container image registry. The image is pulled from the registry and
 embedded in a volume when the virtual machine is created. A
-*registryDisk* volume is ephemeral, and the volume is discarded when
+*containerDisk* volume is ephemeral and is discarded when
 the virtual machine is stopped, restarted, or deleted.
 +
-Registry disks are not limited to a single virtual machine and are
+Container disks are not limited to a single virtual machine, and are
```
Comment (kalexand-rh) [inline modules/cnv_vm_storage_volume_types.adoc]: no comma

**Review by kalexand-rh — COMMENTED** [review]
Couple of minimalism picks, but if it passes SME review and renders well, it overall LGTM

**Comment by vikram-redhat** [comment]
@ousleyp FYI for future.. commits should have been squashed.

**Pull request merged**